Sources/SwiftDEBot/Command/Message Commands/SwiftEvolution.swift

```diff
@@ -0,0 +1,110 @@
+import DiscordBM
+import Foundation
+import FoundationBandAid
+
+struct SwiftEvolutionCommand: MessageCommand {
+    func run(client: DiscordClient, message: Gateway.MessageCreate) async throws {
+        guard message.content.hasPrefix("!se") else { return }
+        let query = message.content.components(separatedBy: " ")[1...].joined(separator: " ")
+        guard query.trimmingCharacters(in: .whitespacesAndNewlines) != "" else {
+            try await client.send(
+                "Ich weiß nicht wonach ich suchen soll. Bitte schreib' eine Nachricht wie `!se async`.",
+                to: message.channel_id)
+            return
+        }
+
+        try await client.setTyping(in: message.channel_id)
+
+        let proposals: [Proposal]
+        do {
+            proposals = try await self.getProposals()
+        } catch {
+            try await client.send("Ich hatte einen Fehler beim Nachschauen 😵 \(error)", to: message.channel_id)
+            return
+        }
+
+        guard proposals.count > 0 else {
+            try await client.send("Auf swift.org sind keine Proposals mehr gelistet 🤔", to: message.channel_id)
+            return
+        }
+        let matchingQuery = proposals.filter { $0.matches(query: query) }
+        guard matchingQuery.count > 0 else {
+            try await client.send("Dazu habe ich leider kein passendes Proposal gefunden 🕵️", to: message.channel_id)
+            return
+        }
+
+        let joinedProposals = matchingQuery.map { $0.shortDescription }.joined(separator: "\n\n")
+        try await client.send(joinedProposals, to: message.channel_id)
+    }
+
+    fileprivate func getProposals() async throws -> [Proposal] {
+        let proposalsURL = URL(string: "https://download.swift.org/swift-evolution/proposals.json")!
```

Comment:
`URLSession` async/await APIs have not been ported to linux unfortunately.
You can implement those APIs yourself, but i recommend just using the `httpClient`:
```swift
let httpClient: HTTPClient = ... /// Pass the httpClient you made for DiscordBM to this function. Don't create a new one.
let url = "https://download.swift.org/swift-evolution/proposals.json"
let response = try await httpClient.execute(.init(url: url), timeout: .seconds(30))
let buffer = try await response.body.collect(upTo: 16 * 1024 * 1024) /// 16 mb
return try JSONDecoder().decode([Proposal].self, from: data)
```
might need some imports like `AsyncHTTPClient`